multiqc/modules/clipandmerge/clipandmerge.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from ClipAndMerge """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import os
+import re
+
+from multiqc import config
+from multiqc.plots import bargraph
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+    """ ClipAndMerge module """
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='ClipAndMerge', anchor='clipandmerge',
+        href="http://www.github.com/apeltzer/ClipAndMerge",
+        info="is a tool for adapter clipping and read merging for ancient DNA data.")
+
+        # Find and load any ClipAndMerge reports
+        self.clipandmerge_data = dict()
+        for f in self.find_log_files('clipandmerge'):
+            self.parse_clipandmerge_log(f)
+
+        # Filter to strip out ignored sample names
+        self.clipandmerge_data = self.ignore_samples(self.clipandmerge_data)
+
+        if len(self.clipandmerge_data) == 0:
+            raise UserWarning
+
+        log.info("Found {} reports".format(len(self.clipandmerge_data)))
+
+        # Write parsed report data to a file
+        self.write_data_file(self.clipandmerge_data, 'multiqc_clipandmerge')
+
+        # Basic Stats Table
+        self.clipandmerge_general_stats_table()
+
+        # Alignment Rate Plot
+        self.add_section( plot = self.clipandmerge_alignment_plot() )
```

Comment:
I should have said - both here and in the other module; it's good to add helptext and other stuff here to help people interpret their data. See http://multiqc.info/docs/#step-5-create-report-sections

Reply:
Done - see 34d7895

Reply:
Ah - yes but that went in the wrong place sorry. You added that to the plot config, this was as arguments for the `add_section` function. To be honest, given that the text basically repeats what's written there anyway I think it's fine to just drop them here. The stuff I would put in the help text is a longer block text explaining what the different categories mean for example, perhaps with a mention of what can be expected from typical runs.

The description is more used when modules have multiple sections and isn't really required here.